Tasks/ANT/ant.ps1

```diff
@@ -245,40 +205,43 @@ else
 # check if code coverage has been enabled
 if($isCoverageEnabled)
 {
-	
-   # run report code coverage task which generates code coverage reports.
-   $reportsGenerationFailed = $false
-   Write-Verbose "Collecting code coverage reports" -Verbose
-   try
-   {
-	if(Test-Path $reportBuildFile)
-	{
-		# This will handle compat between S91 and S92
-		Invoke-Ant -AntBuildFile $reportBuildFile -Targets $CCReportTask
-	}
-	else
-	{
-		Invoke-Ant -AntBuildFile $antBuildFile -Targets $CCReportTask
-	}
-   }
-   catch
-   {
-	$reportsGenerationFailed = $true
-   }
+    
+    # run report code coverage task which generates code coverage reports.
+    $reportsGenerationFailed = $false
+    Write-Verbose "Collecting code coverage reports" -Verbose
+    try
+    {
+        if(Test-Path $reportBuildFile)
+        {
+            # This will handle compat between S91 and S92
+            Invoke-Ant -AntBuildFile $reportBuildFile -Targets $CCReportTask
+        }
+        else
+        {
+            Invoke-Ant -AntBuildFile $antBuildFile -Targets $CCReportTask
+        }
+    }
+    catch
+    {
+        $reportsGenerationFailed = $true
+    }
    
-	
-	if(-not $reportsGenerationFailed -and (Test-Path $summaryFile))
-   	{
-		Write-Verbose "Summary file = $summaryFile" -Verbose
-		Write-Verbose "Report directory = $reportDirectory" -Verbose
-		Write-Verbose "Calling Publish-CodeCoverage" -Verbose
-		Publish-CodeCoverage -CodeCoverageTool $codeCoverageTool -SummaryFileLocation $summaryFile -ReportDirectory $reportDirectory -Context $distributedTaskContext    
-   	}
-   	else
-   	{
-        	Write-Host "##vso[task.logissue type=warning;code=006003;]"
-		Write-Warning "No code coverage results found to be published. This could occur if there were no tests executed or there was a build failure. Check the ant output for details." -Verbose
-   	}
+    if(-not $reportsGenerationFailed -and (Test-Path $summaryFile))
+       {
+        Write-Verbose "Summary file = $summaryFile" -Verbose
+        Write-Verbose "Report directory = $reportDirectory" -Verbose
+        Write-Verbose "Calling Publish-CodeCoverage" -Verbose
+        Publish-CodeCoverage -CodeCoverageTool $codeCoverageTool -SummaryFileLocation $summaryFile -ReportDirectory $reportDirectory -Context $distributedTaskContext    
+       }
+       else
+       {
+        Write-Host "##vso[task.logissue type=warning;code=006003;]"
+        Write-Warning "No code coverage results found to be published. This could occur if there were no tests executed or there was a build failure. Check the ant output for details." -Verbose
+       }
+
+    # Reset temp copy and file permissions are reset by default
+    Copy-Item "$antBuildFile.tmp" $antBuildFile -Force -ErrorAction Continue
+    Remove-Item "$antBuildFile.tmp" -Force -ErrorAction Continue
```

Comment:
eveywhere else also

---
In reply to: [73139119](https://github.com/Microsoft/vsts-tasks/pull/2184#discussion_r73139119) [](ancestors = 73139119)